test/common/network/listen_socket_impl_test.cc

```diff
@@ -126,6 +126,15 @@ class ListenSocketImplTest : public testing::TestWithParam<Address::IpVersion> {
     EXPECT_GT(socket->addressProvider().localAddress()->ip()->port(), 0U);
     EXPECT_EQ(Type, socket->socketType());
   }
+
+  void testLowerCaseRequestedServerName() {
```

Comment:
This function is not invoked during the test
You probably want something like 
```
TEST_P(ListenSocketImplTestTcp, TestLowerCaseRequestedServerName) {
         Your code here.
}

Reply:
got it, thanks